Dockerfile

```diff
@@ -34,5 +35,5 @@ COPY . .
 # COPY .env.example ./.env
 
 WORKDIR public
-CMD ["php", "-S", "0.0.0.0:8000"]
-EXPOSE 8000
\ No newline at end of file
+CMD ["php", "-S", "0.0.0.0:80"]
+EXPOSE 80
```

Comment:
Any particular reason for the port change?  Simpler use?  My selection was also arbitrary.

Reply:
So, it's http server. The ONLY http server that will ever run on this container. The port for http server is 80.
As a practical reason, you would like this image to be runnable with zero configuration on services like render.com. Having 80 as default is most reasonable thing.

Reply:
My mistake, I was thinking about the exposed port -- the docker-compose file had `ports: 8000:8000` for the host mapping.

This Dockerfile uses `EXPOSE` to make the service accesible on port 80, http -- I agree that it's reasonable, but I wasn't sure if it was a good idea as users might (????) have something else running that uses that port.  Your choice of port 80 is probably the best first pass.  Thanks!